packages/hash/frontend/src/components/layout/PageSidebar/NavLink.tsx

```diff
@@ -1,63 +1,102 @@
-import { useState, FC, ReactNode } from "react";
-import { faChevronRight } from "@fortawesome/free-solid-svg-icons";
-import { Box, Typography, Collapse } from "@mui/material";
+import { useState, FC } from "react";
+import { faAdd, faChevronRight } from "@fortawesome/free-solid-svg-icons";
+import { Box, Typography, Collapse, Tooltip } from "@mui/material";
 import { FontAwesomeIcon } from "../../icons";
-import { IconButton } from "../../IconButton";
-// import { Link } from "./Link";
+import { IconButton, IconButtonProps } from "../../IconButton";
 
 type NavLinkProps = {
   title: string;
-  endAdornment: ReactNode;
+  endAdornmentProps: {
+    tooltipTitle: string;
+    "data-testid"?: string;
+  } & IconButtonProps;
 };
 
 export const NavLink: FC<NavLinkProps> = ({
   title,
   children,
-  endAdornment,
+  endAdornmentProps,
 }) => {
   const [expanded, setExpanded] = useState(true);
+  const [hovered, setHovered] = useState(false);
+  const {
+    tooltipTitle: endAdornmentTooltipTitle,
+    sx: endAdormentSx = [],
+    ...otherEndAdornmentProps
+  } = endAdornmentProps;
 
   return (
     <Box>
       <Box
-        sx={{
+        sx={({ palette }) => ({
           display: "flex",
           alignItems: "center",
-          py: "9px",
-          pl: "18px",
-          pr: 0.5,
+          borderRadius: "4px",
+          py: "8px",
+          pl: "12px",
+          pr: "6px",
           mx: 0.5,
-        }}
+          ...(hovered && {
+            backgroundColor: palette.gray[20],
+          }),
+        })}
+        onMouseOver={() => setHovered(true)}
+        onMouseOut={() => setHovered(false)}
       >
         <Typography
           variant="smallCaps"
-          sx={{
+          sx={({ palette }) => ({
             mr: 1.4,
-            color: ({ palette }) => palette.gray[50],
-          }}
+            color: palette.gray[50],
+          })}
         >
           {title}
         </Typography>
         <IconButton
-          size="small"
+          size="xs"
           unpadded
           rounded
-          sx={{
+          sx={({ palette }) => ({
             mr: "auto",
-          }}
+            ...(!expanded && { color: palette.gray[40] }),
+            ...(hovered && {
+              backgroundColor: palette.gray[30],
+              color: palette.gray[80],
+            }),
+          })}
           onClick={() => setExpanded((prev) => !prev)}
         >
           <FontAwesomeIcon
-            sx={{
+            sx={({ transitions }) => ({
               transform: expanded ? `rotate(90deg)` : "none",
-              transition: ({ transitions }) =>
-                transitions.create("transform", { duration: 300 }),
-            }}
+              transition: transitions.create("transform", { duration: 300 }),
```

Comment:
```suggestion
              transition: transitions.create("transform"),
```

Reply:
Fixed here => https://github.com/hashintel/hash/pull/409/commits/a2c54ce0bd5494d78dfa3935a2647fc8fa5fe0d6#diff-6e1277ecccf8b78ca260fe2e4f528995462d5f0c10df1c5e329ba9b9ad87b809

Reply:
Doesn't block merging, but this didn't get addressed in that commit (or other commits after that).

Reply:
Sorry I missed that. Fixed here => [b11b8e2](https://github.com/hashintel/hash/pull/409/commits/b11b8e28b108b73900c7544304454ca099a95866)

Reply:
No worries, thanks